WebApplication/ClientApp/src/actions/adoptWithParamsActions.js

```diff
@@ -20,26 +20,36 @@ export const adoptProjectWithParameters = (parameters) => async (dispatch) => {
                 dispatch(addLog('JobManager: HubConnection started for adopt project with params'));
             },
             // onComplete
-            (tuple) => {
+            (projectwithparams) => {
```

Comment:
please use [camelCase](https://github.com/airbnb/javascript#naming--camelCase)

Reply:
Fixed in last push, thank you Eugene :)